coalib/bearlib/languages/definitions/SASS.py

```diff
@@ -0,0 +1,11 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class SASS:
+    extensions = '.sass',
+    comment_delimiters = '//',
+    multiline_comment_delimiters = {'/*': '*/'}
+    encapsulators = {'(': ')', '[': ']'}
+    versions = {3.0, 3.1, 3.2, 3.3, 3.4, 3.5}
```

Comment:
SASS also has versions 3.6 and 4.0. [Here](https://rubygems.org/gems/sass/versions)

Reply:
Those aren't official or stable yet, I guess. Leave it